### DISCUSSION TITLE
Pufferfish Mincraft egg

### DISCUSSION THREAD

**Discussion by crederedeus** [pterodactyl/panel#4066] [Feature Requests]
This egg has been requested several times, so I offer it for distribution. It is Pufferfish, a Fork from PaperSpigot, for the Minecraft game. The parameters have been optimized.

You can import it directly from your panel.
File : https://file.io/37ERKiMgVj3Y

Thanks

**Comment by Technofied** [comment]
"The transfer you requested has been deleted."

**Comment by leeteral** [comment]
bump
